src/main/java/org/cactoos/io/CheckedBytes.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.io;
+
+import java.io.IOException;
+import org.cactoos.Bytes;
+import org.cactoos.Func;
+import org.cactoos.scalar.CheckedScalar;
+
+/**
+ * Bytes that throws exception of specified type.
+ *
+ * @author Roman Proshin (roman@proshin.org)
+ * @version $Id$
+ * @param <E> Exception's type.
+ * @since 1.0
```

Comment:
@proshin-roman Why do you use `1.0` here? Looks like the next release will be `0.31`

Reply:
@SharplEr that's because of the current pom.xml which has 1.0-SNAPSHOT version (https://github.com/yegor256/cactoos/blob/master/pom.xml#L34). But you're right that according to the latest version that is 0.30 the next release should be 0.31. So, I will fix version in the pull request and submit a bug about incorrect pom.xml version.